frontend/public/components/nav/index.tsx

```diff
@@ -22,6 +22,7 @@ export const Navigation: React.FC<NavigationProps> = React.memo(
       }
       isNavOpen={isNavOpen}
       theme="dark"
+      style={{ minHeight: '100%' }}
```

Comment:
why do we need to override the style here?

Reply:
adding the app in the container for drawer content caused the sidebar to not takeup the full height leaving a blank space. this is the fix for that

Reply:
Looks like this breaks layout adding extra scrollbars and giving too much room to pages like topology.
![image](https://user-images.githubusercontent.com/14068621/86603169-aa718f00-bf71-11ea-8c08-5bd4219f02ed.png)

Reply:
This is still an issue.

Reply:
Fixed by issue: https://github.com/patternfly/patternfly/pull/3268

For now we can override this style in console by doing this:
```
// TODO remove this override once https://github.com/patternfly/patternfly/pull/3268 is adopted
.pf-c-drawer .pf-c-page__main {
  min-height: auto !important;
}
```